src/components/CreateAccountPage.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Button, FormControl, FormGroup } from 'react-bootstrap'
+import { Link } from 'react-router-dom'
+
+/* global fetch */
+
+export default class LogInPage extends Component {
+  constructor (props) {
+    super(props)
+
+    this.getValidationState = this.getValidationState.bind(this)
+    this.checkValidationState = this.checkValidationState.bind(this)
+    this.handleLogInClick = this.handleLogInClick.bind(this)
+    this.createAccount = this.createAccount.bind(this)
+  }
+
+  getValidationState () {
+    let checkInfo = this.props.accountName
+
+    let found = Object.keys(this.props.dbAccounts).findIndex((name) => {
+      return name === checkInfo
+    })
+
+    if (found !== -1 || checkInfo.length < 3) {
+      return 'warning'
+    } else if (found === -1) {
+      return 'success'
+    }
+  }
+
+  checkValidationState () {
+    if (this.getValidationState() === 'warning') {
+      return true
+    } else {
+      return false
+    }
+  }
```

Comment:
this could be written as simply:
`return this.getValidattionState() === WARNING_INDICATOR`